docs/surveys/salary-survey/2022.rst

```diff
@@ -0,0 +1,2208 @@
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+   | Human Resources, Recruitment, Careers, Jobs                                                                                        | 7   | 0.7%       |
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+   | Agriculture                                                                                                                        | 6   | 0.6%       |
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+   | Travel, Hospitality, Holidays, Hotels, Accommodation                                                                               | 5   | 0.5%       |
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+   | Non-profit, Community                                                                                                              | 5   | 0.5%       |
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+   | Events, Event Management, Event Services, Venues, Audio/Video                                                                      | 4   | 0.4%       |
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+   | Real Estate                                                                                                                        | 3   | 0.3%       |
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+   | Legal Services                                                                                                                     | 3   | 0.3%       |
+   +------------------------------------------------------------------------------------------------------------------------------------+-----+------------+
+
+Location
+--------
+
+.. raw:: html
+   
+   <details><summary>What we asked</summary>
+
+.. container:: question
+   
+   In which country is your organization based?
+   
+   - State, Province, Territory or Region
+   - City or Town
+
+.. raw:: html
+
+   </details>
+
+47.2% of all organizations employing survey respondents in 2022 were based in North America, with the vast majority of those in the United States (43.8% of the total, versus 3.4% for Canada). Global or Multi-national organizations accounted for the second largest proportion, 25.7%. European countries account for another 20%.
```

Comment:
typo: we use Multinational later in the table, which I prefer to "Multi-national"